notebooks/113-image-classification-quantization/113-image-classification-quantization.ipynb

```diff
@@ -0,0 +1,446 @@
+        "    def __init__(self, top_k=1):\n",
+        "        super().__init__()\n",
+        "        self._top_k = top_k\n",
+        "        self._name = 'accuracy@top{}'.format(self._top_k)\n",
+        "        self._matches = []\n",
+        "\n",
+        "    @property\n",
+        "    def value(self):\n",
+        "        \"\"\" Returns accuracy metric value for the last model output. \"\"\"\n",
+        "        return {self._name: self._matches[-1]}\n",
+        "\n",
+        "    @property\n",
+        "    def avg_value(self):\n",
+        "        \"\"\" Returns accuracy metric value for all model outputs. \"\"\"\n",
+        "        return {self._name: np.ravel(self._matches).mean()}\n",
+        "\n",
+        "    def update(self, output, target):\n",
+        "        \"\"\" Updates prediction matches.\n",
+        "        :param output: model output\n",
+        "        :param target: annotations\n",
+        "        \"\"\"\n",
+        "        if len(output) > 1:\n",
+        "            raise Exception('The accuracy metric cannot be calculated '\n",
+        "                            'for a model with multiple outputs')\n",
+        "        if isinstance(target, dict):\n",
+        "            target = list(target.values())\n",
+        "        predictions = np.argsort(output[0], axis=1)[:, -self._top_k:]\n",
+        "        match = [float(t in predictions[i]) for i, t in enumerate(target)]\n",
+        "\n",
+        "        self._matches.append(match)\n",
+        "\n",
+        "    def reset(self):\n",
+        "        \"\"\" Resets collected matches \"\"\"\n",
+        "        self._matches = []\n",
+        "\n",
+        "    def get_attributes(self):\n",
+        "        \"\"\"\n",
+        "        Returns a dictionary of metric attributes {metric_name: {attribute_name: value}}.\n",
+        "        Required attributes: 'direction': 'higher-better' or 'higher-worse'\n",
+        "                             'type': metric type\n",
+        "        \"\"\"\n",
+        "        return {self._name: {'direction': 'higher-better',\n",
+        "                             'type': 'accuracy'}}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CclWk-fVd9Wi"
+      },
+      "source": [
+        "## Run Quantization Pipeline and compare the accuracy of the original and quantized models\n",
+        "Here we define a configuration for our quantization pipeline and run it. \n",
+        "\n",
+        "**Note**: we use built-in `IEEngine` implementation of the `Engine` interface from the POT API for model inference. `IEEngine` is built on top of OpenVINO Python* API for inference and provides basic functionality for inference of simple models. If you have a more complicated inference flow for your model/models you should create your own implementation of `Engine` interface, for example by inheriting from `IEEngine` and extending it."
```

Comment:
This is a simple detail, but we would love to see constancy in the word format. May you please define the format for words, such as "Note". This one has different formats throughout the notebook. Thanks.